src/js/video.js

```diff
@@ -7,6 +7,8 @@
 
 import window from 'global/window';
 import document from 'global/document';
+import * as browser from './utils/browser.js';
```

Comment:
this needs to be the first local import, otherwise, rollup puts Dom below browser in the output file and it breaks at run time.